frontend/src/pull.ts

```diff
@@ -187,6 +187,10 @@ export class Pull extends PullData {
     const linkIdPrefix = isReview ? "pullrequestreview" : "issuecomment";
     return this.url() + `#${linkIdPrefix}-${sig.data.comment_id}`;
   }
+
+  getUrlOfBaseBranch() {
+    return "https://github.com/" + this.repo + "/compare/" + this.base.ref;
```

Comment:
Sorrry, one more thing: 
```suggestion
    return "https://github.com/" + this.repo + "/compare/" + encodeURIComponent(this.base.ref);
```

branch names can contain all sorts of non-url-friendly chars.